tests/update/patch_before_migration.pm

```diff
@@ -20,7 +20,7 @@ sub is_smt_or_module_tests {
 }
 
 sub system_prepare() {
-    select_console 'root-console';
+    select_console('root-console');
```

Comment:
why change? I think we should avoid reworking unrelated sections when it's not adding real value